src/schemars.rs

```diff
@@ -0,0 +1,21 @@
+pub use schemars::{JsonSchema, gen::SchemaGenerator, schema::{Schema, SchemaObject, InstanceType}};
+
+use crate::{FixedPoint};
+
+impl<I, P> JsonSchema for FixedPoint<I, P> {
+    fn is_referenceable() -> bool {
+        false
+    }
+
+    fn schema_name() -> String {
+        "String".to_owned()

```

Comment:
Why not `FixedPoint` here?

Reply:
Ideally, it should be `FixedPoint(<i>, <p>)`, but I'm not sure whether it's acceptable by all generators or not, so `FixedPoint` is good enough for now.

Reply:
renamed